### ISSUE TITLE
Mistake with resize event handler

### ISSUE THREAD

**Issue by ArtemFitiskin** [manifestinteractive/jqvmap#129]
Hello!

Look at this fragment in your plugin source code:

```javascript
jQuery(window).resize(function () {
	// ...
});
```

If page is loaded dynamically or a node for the map is loaded with ajax, every time `vectorMap` is initialized, `window` binds a new resize event handler. These events are accumulated and are executed simultaneously, causing the error [e.g.](https://cloud.githubusercontent.com/assets/5507542/4489410/ae9622d0-4a1a-11e4-9ca9-c1f2aecf939f.png) (in case of resize - you get an error with division by zero because `params.container.width()` is 0, and the old element has been removed from DOM). 

Also, you need to unbind old `.jqvmap` event namespace when initializing the plugin.

**Comment by manifestinteractive** [comment]
@ArtemFitiskin This is now resolved and ready for testing in https://github.com/manifestinteractive/jqvmap/tree/issue-129

Fix unbinds events on init and checks for valid width & height before applying them.  Also added a new `onResize` event in the docs to capture new map size when it is resized.  I also added unit tests for the onResize event handler.

If wanted to give you a chance to test this before I merged it into master, which I will do in a few days if I do not hear anything.
